docs/createInputsReducer.md

```diff
@@ -0,0 +1,62 @@
+## createInputsReducer(inputsConfig:Object)
+
+You use this create reducers for your forms. You need one reducer per form. 
+
+Example:
+
+    import { createStore, combineReducers } from 'redux';
+    import { reducer as formReducer } from 'redux-form';
+    
+    const inputsConfig = {
+        ... redux-inputs configuration ...
+    };
+    
+    const reducers = {
+        inputs: createInputsReducer(inputsConfig)
+    };
+    
+    const reducer = combineReducers(reducers);
+    const store = createStore(reducer);
+    
+### Use a different redux mount point
+
+By default, redux-inputs assumes the reducer is mounted at `inputs`. For mounting in a different location, or 
+when multiple forms are stored in the state, use this special property in your inputsConfig object:
+
+#### `_reduxMountPoint` *(String)* change 
+
+    inputsConfig = {
+        _reduxMountPoint: 'tab1.inputs'
+        ... other inputs config ...
+    };
+    
+In this example your state would look like this:
+
+    {
+        tab1: {
+            inputs: {}
+        }
+    }
+
+### Multiple forms
+
+Create an inputsConfig for each form, making sure that each have a unique `_reduxMountPoint`.
```

Comment:
It would be nice to explain why this is necessary. For examples, something like: "To avoid collisions at the default mount point, specify a custom mount point when using multiple forms."